Extension/i18n/fra/package.i18n.json

```diff
@@ -419,15 +422,15 @@
 	"c_cpp.walkthrough.description": "Permet de découvrir la riche expérience de développement C++ de VS Code.",
 	"c_cpp.walkthrough.set.up.title": "Configurer votre environnement C++",
 	"c_cpp.walkthrough.activating.description": "Activation de l’extension C++ pour déterminer si votre environnement C++ a été configuré.\nActivation de l’extension...",
-	"c_cpp.walkthrough.no.compilers.windows.description": "Nous n’avons pas trouvé de compilateur C++ sur votre machine, ce qui est nécessaire pour utiliser l’extension C++. Suivez les instructions de droite pour en installer un, puis cliquez sur « Rechercher mon nouveau compilateur » ci-dessous.\n[Rechercher mon nouveau compilateur](command:C_Cpp.RescanCompilers?%22walkthrough%22)",
-	"c_cpp.walkthrough.no.compilers.description": "Nous n’avons pas trouvé de compilateur C++ sur votre machine, ce qui est nécessaire pour utiliser l’extension C++. Sélectionnez « Installer un compilateur C++ » pour installer un compilateur pour vous ou suivez les instructions à droite pour en installer un, puis cliquez sur « Rechercher mon nouveau compilateur » ci-dessous.\n[Installer un compilateur C++](command:C_Cpp.InstallCompiler?%22walkthrough%22)\n[Rechercher mon nouveau compilateur](command:C_Cpp.RescanCompilers?%22walkthrough%22)",
-	"c_cpp.walkthrough.compilers.found.description": "L’extension C++ fonctionne avec un compilateur C++. Sélectionnez-en un parmi ceux déjà présents sur votre ordinateur en cliquant sur le bouton ci-dessous.\n[Sélectionner mon compilateur par défaut](command:C_Cpp.SelectIntelliSenseConfiguration?%22walkthrough%22)",
+	"c_cpp.walkthrough.no.compilers.windows.description": "Nous n’avons pas trouvé de compilateur C++ sur votre machine, ce qui est nécessaire pour utiliser l’extension C++. Suivez les instructions de droite pour en installer un, puis cliquez sur « Rechercher mon nouveau compilateur » ci-dessous.\n[Rechercher mon nouveau compilateur](command:C_Cpp.RescanCompilers ?%22walkthrough%22)",
```

Comment:
Remove these spaces in the commands (multiple instances)

Reply:
Fixed -- I re-activated the internal bug on this.